user-guide/Troubleshooting/Procedures/Remote_Access_Issues.md

```diff
@@ -0,0 +1,21 @@
+---
+uid: Remote_Access_Issues
+---
+
+# Remote Access Issues
+Follow the guide below when you're encountering connections issues with dataminer.services features and your DataMiner System.
+
+## Check Status of DxMs
+A common cause for issues with [dataminer.services](https://docs.dataminer.services/user-guide/Cloud_Platform/AboutCloudPlatform/About_the_Cloud_Platform.html?q=dataminer.services) features and your DMS is due to the connection from the [CloudGateway](https://docs.dataminer.services/user-guide/Reference/DataMiner_Extension_Modules.html?q=dxm#cloudgateway) towards dataminer.services being unavailable. 
```

Comment:
use xref with the page uid instead of url, in docs you can refer to another page like this so you don't have broken urls when moving files around